<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MMT-LPT-5603N" tests="12" failures="3" timestamp="1 Oct 2019 18:39:32 GMT" time="1.296" errors="0">
  <testcase name="FailedTest" time="0.018" classname="ListenerExample">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at ListenerExample.FailedTest(ListenerExample.java:27)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- FailedTest -->
  <testcase name="passedTEst" time="0.0" classname="ListenerExample"/>
  <testcase name="passedTEst2" time="0.0" classname="ListenerExample"/>
  <testcase name="readExcel" time="1.18" classname="ListenerExample"/>
  <testcase name="FailedTest" time="0.0" classname="ListenerExample">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at ListenerExample.FailedTest(ListenerExample.java:27)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- FailedTest -->
  <testcase name="passedTEst" time="0.0" classname="ListenerExample"/>
  <testcase name="passedTEst2" time="0.001" classname="ListenerExample"/>
  <testcase name="readExcel" time="0.036" classname="ListenerExample"/>
  <testcase name="FailedTest" time="0.0" classname="ListenerExample">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at ListenerExample.FailedTest(ListenerExample.java:27)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- FailedTest -->
  <testcase name="passedTEst" time="0.0" classname="ListenerExample"/>
  <testcase name="passedTEst2" time="0.0" classname="ListenerExample"/>
  <testcase name="readExcel" time="0.034" classname="ListenerExample"/>
</testsuite>
